### PR TITLE
Add Download links to the beta version

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: content/pages/download.md
Comment (Swiftb0y) [inline content/pages/download.md]: I think we should emphasize that we prefer the beta snapshot being tested over the development snapshot.

Comment (daschuer) [reply content/pages/download.md]: Yes, currently both texts are kind of duplicated. Not sure if it woks if we short the text for the alpha to a minimum. 
Ideas?

Comment (Swiftb0y) [reply content/pages/download.md]: How about we imply that testing the beta is usually preferred? something like this:

> If you're bored of thoroughly testing the beta snapshots - or there may not be a beta running currently - you can also test the current development snapshot to get a glimpse at the even newer features. As for the beta snapshots, please refer to the [Testing wiki page](https://github.com/mixxxdj/mixxx/wiki/Testing) for where to find the latest builds and instructions how to test pull requests before they are merged.

**Comment by daschuer** [comment]
I have incoperate you suggestions.
From now we will always have a beta. If we have no 2.5 beta after 2.4.0 is released. The link will still point to the Head of the 2.4 branch which is IMHO OK for a few days preparing the 2.5 beta.

File: content/pages/download.md
Comment (ronso0) [inline content/pages/download.md]:
```suggestion
      A great way to contribute to Mixxx is testing the upcoming version before it is being released.
```

File: content/pages/download.md
Comment (ronso0) [inline content/pages/download.md]:
```suggestion
      This is the only way to find bugs in time before a release that may have slipped through the initial review process or affect rare use cases.
```

**Comment by daschuer** [comment]
Thank you.

File: content/pages/download.md
Comment (Swiftb0y) [inline content/pages/download.md]:
```suggestion
    title: Beta Snapshots (2.4)
```

**Review by Swiftb0y — APPROVED** [review]
LGTM otherwise. waiting for LGTM from @ronso0.

**Review by ronso0 — APPROVED** [review]
LGTM (with the "(2.4)" appended)

**Comment by daschuer** [comment]
Done. Thank you. 

After merge we may start with the series of teaser blogs. Maybe we can find a new contributor for that. We can really make use of people that have fun to juggle with words instead of code.

**Comment by ronso0** [comment]
Nice, thank you!

**Comment by Swiftb0y** [comment]
I'd volunteer for now, at least until we find someone else. Just gotta find the time.

**Pull request merged**
